src/components/bs5/sidenav/sidenav.scss

```diff
@@ -93,6 +106,7 @@
                 font-size: 0.875rem;
                 color: var(--body-color);
                 text-decoration: none;
+                transition-duration: 0;
 
                  //li a.nav-link.active
                 &.active {
```

Comment:
This rule is on `.nav-link` but template is printing active class on `.nav-item` element.

Active item text should be `color: #04284C`, and I think shouldn't have any link/hover interactions (as it would be linking back to the current page) ?

Reply:
I've restructured slightly, active elements are now span's, they don't need to be links.

I've added a --sidenav-active-item-color variable, uses heading text HEX for light and dark modes.